index.js

```diff
@@ -217,6 +217,10 @@ module.exports = class Hyperdrive extends ReadyResource {
     return this.files.del(normalizePath(name))
   }
 
+  compare (a, b) {
+    return a.seq - b.seq
```

Comment:
I think it should return `-1`, `0`, or `1`, then if Localdrive does the same then both API results are the same

Reply:
I just don't know if you actually need the difference in the output?

Reply:
sure, up to you